manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.WorldMap
+PackageVersion: 1.4.0
+PackageLocale: en-US
+Publisher: Simon G4ELI

```

```suggestion
Publisher: Simon Brown, G4ELI
```
Should match control panel